var/spack/repos/builtin/packages/py-rdflib/package.py

```diff
@@ -21,15 +21,23 @@ class PyRdflib(PythonPackage):
     homepage = "https://github.com/RDFLib/rdflib"
     pypi = "rdflib/rdflib-5.0.0.tar.gz"
 
+    version("6.3.2", sha256="72af591ff704f4caacea7ecc0c5a9056b8553e0489dd4f35a9bc52dbd41522e0")
     version("6.2.0", sha256="62dc3c86d1712db0f55785baf8047f63731fa59b2682be03219cb89262065942")
     version("6.0.2", sha256="6136ae056001474ee2aff5fc5b956e62a11c3a9c66bb0f3d9c0aaa5fbb56854e")
     version("5.0.0", sha256="78149dd49d385efec3b3adfbd61c87afaf1281c30d3fcaf1b323b34f603fb155")
 
-    depends_on("python@3.7:", when="@6:", type="build")
-    depends_on("py-setuptools", type="build")
+    depends_on("python@3.7:3", when="@6.3:", type=("build", "run"))
+    depends_on("py-poetry-core@1.4:", when="@6.3:", type="build")
 
+    depends_on("py-isodate@0.6:0", when="@6.3:", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("py-isodate@0.6", when="@6.3:", type=("build", "run"))
```
See https://python-poetry.org/docs/dependency-specification/#caret-requirements

Reply:
Thanks